language/tools/move-cli/src/lib.rs

```diff
@@ -27,6 +28,13 @@ use std::{
     path::{Path, PathBuf},
 };
 
+pub const MOVE_SRC: &str = "move_src";
+pub const MOVE_DATA: &str = "move_data";
+/// Store modules under move_src without an explicit `address {}` block under 0x2.
```

Comment:
I plan on just removing this feature really... 
Just haven't had the time due to the amount of tests that need to be changed

Reply:
Ah, gotcha. I will try to avoid using it in tests to decrease the pain for later...